modules/installing-gitops-operator-in-web-console.adoc

```diff
@@ -10,10 +10,45 @@ You can install {gitops-title} Operator from the OperatorHub by using the web co
 
 .Procedure
 
-. Open the *Administrator* perspective of the web console and navigate to *Operators* → *OperatorHub* in the menu on the left.
+. Open the *Administrator* perspective of the web console and go to *Operators* -> *OperatorHub*.
 
 . Search for `OpenShift GitOps`, click the *{gitops-title}* tile, and then click *Install*.
+
+. On the *Install Operator* page:
+
+.. Select an *Update channel*.
+
+.. Select a {gitops-shortname} *Version* to install.
+
+.. Choose an *Installed Namespace*. The default installation namespace is `openshift-gitops-operator`.
++
+[WARNING]
+====
+For the {gitops-shortname} version 1.10 and later, the default namespace changed from `openshift-gitops` to `openshift-gitops operator`. If the Operator is installed in a namespace different from `openshift-gitops operator` and you update it from version 1.9 to version 1.10, it can result in unexpected behavior.
```

Comment:
The default namespace changed from `openshift-operators` to `openshift-gitops-operator`, it was never installed in `openshift-gitops` namespace

Also, upgrading operators in a different namespace should not cause any issues. The only known issue was that of an alert being generated from a mismatch in service monitor configuration on upgrade, but that should be fixed in 1.11.0

Reply:
@jaideepr97 I see, I did not even notice that I included `openshift-gitops` instead of `openshift-operators`,  thank you for noticing! Also, since this is being fixed in 1.11 (and 1.10.2 as I see in the Jira that tracks the issue) I think it would be best to just include this as a note:
```For the GitOps version 1.10 and later, the default namespace changed from `openshift-operators` to `openshift-gitops operator`.``` WDYT?

Reply:
Sounds good